ipaserver/install/httpinstance.py

```diff
@@ -305,12 +316,22 @@ def configure_certmonger_renewal_guard(self):
             if certmonger_stopped:
                 certmonger.stop()
 
-    def create_cert_db(self):
-        database = certs.NSS_DIR
-        pwd_file = os.path.join(database, 'pwdfile.txt')
+    def create_cert_dbs(self):
+        self.create_cert_db(paths.HTTPD_ALIAS_DIR, constants.HTTPD_USER)
+        self.create_cert_db(paths.IPA_RADB_DIR, constants.IPAAPI_USER)
```

Comment:
I don't think `HTTPInstance` has any bussiness in creating the RA DB, it should in fact be done in `CAInstance`.

Also it shouldn't be created in CA-less mode.